apps/web/src/lib/stores/index.ts

```
@@ -1,19 +1,22 @@
-import { api } from "@buzztrip/backend/api";
 import { Id } from "@buzztrip/backend/dataModel";
 import type { CombinedMarker, Map } from "@buzztrip/backend/types";
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)